.github/workflows/collector.yml

```
@@ -29,12 +29,13 @@ env:
 
 jobs:
   build-collector-image:
-    name: Build Collector
-    runs-on: ubuntu-24.04
+    name: Local collector image ${{ matrix.arch }}
     strategy:
-      fail-fast: false
       matrix:
-        arch: [amd64, ppc64le, arm64]
+        arch: [amd64, arm64, ppc64le]
+        exclude:
+          - arch: ${{ (!contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds') && 'ppc64le') || '' }}
```

Same comment as in the builder workflow
```suggestion
          - arch: ${{ ((
            github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds')
          ) && 'ppc64le') || '' }}
```